### PR TITLE
ADEN-3411 Add bottom leaderboard to Oasis

### PR DESCRIPTION
**Pull request by drets**
- new slot BOTTOM_LEADERBOARD (direct, remnant, fastlane providers, fastlane bidder)
- sizes: 728x90, 970x250, 1024x416
- remove SKY3 hack for BFAB
- remove Karma warnings

### CODE REVIEW

File: skins/oasis/css/core/ads.scss [hunk 1]
```diff
@@ -184,16 +184,13 @@ $taboola-small-breakpoint: "#{$screen} and (max-width: 420px)";
...
-		padding: 13px 0 23px 0;
-		top: 0 !important;
-		width: 100%;
-	}
+#BOTTOM_LEADERBOARD {
+	overflow: hidden;
+	padding: 13px 0 23px 0;
+	position: relative;
+	text-align: center;
+	top: 0 !important;
```
Comment (fraszczakszymon) [inline skins/oasis/css/core/ads.scss]: Not needed

File: tests/karma/js-unit.conf.js [hunk 2]
```diff
@@ -99,8 +99,6 @@ module.exports = function (config) {
 			//ARecoveryEngine
 			'extensions/wikia/ARecoveryEngine/js/*.js',
 			'extensions/wikia/ARecoveryEngine/js/**/*.js',
-			'extensions/wikia/ARecoveryEngine/js/spec/**/*.spec.js',
```
Comment (fraszczakszymon) [inline tests/karma/js-unit.conf.js:102]: Why those lines are removed?

Comment (drets) [inline tests/karma/js-unit.conf.js:102]: To remove Karma warnings.

File: skins/oasis/css/core/ads.scss [hunk 3]
```diff
@@ -184,16 +184,13 @@ $taboola-small-breakpoint: "#{$screen} and (max-width: 420px)";
...
-		overflow: hidden;
-		position: relative;
-		padding: 13px 0 23px 0;
-		top: 0 !important;
-		width: 100%;
-	}
+#BOTTOM_LEADERBOARD {
+	overflow: hidden;
+	padding: 13px 0 23px 0;
+	position: relative;
```
Comment (fraszczakszymon) [inline skins/oasis/css/core/ads.scss]: Not needed

**Comment by fraszczakszymon** [comment]
LGMT 👍

**Pull request merged**